k8s.gcr.io/manifests/k8s-staging-test-infra/promoter-manifest.yaml

```diff
@@ -1,14 +1,50 @@
 # google group for gcr.io/k8s-staging-test-infra is k8s-infra-staging-test-infra@kubernetes.io
-
 registries:
 - name: gcr.io/k8s-staging-test-infra
   src: true
-
-# Promotes to the following GCR locations:
-# - {us,eu,asia}.gcr.io/k8s-test-infra --> k8s.gcr.io
-- name: us.gcr.io/k8s-test-infra
-  service-account: k8s-infra-gcr-promoter@k8s-test-infra.iam.gserviceaccount.com
-- name: eu.gcr.io/k8s-test-infra
-  service-account: k8s-infra-gcr-promoter@k8s-test-infra.iam.gserviceaccount.com
-- name: asia.gcr.io/k8s-test-infra
-  service-account: k8s-infra-gcr-promoter@k8s-test-infra.iam.gserviceaccount.com
```

Comment:
This manifest was promoting to a non-existent destination. @BenTheElder, are you happy with the rename?

```
 REDACTED  MCW0CDP3YY  tmp  test  $  gcrane ls us.gcr.io/k8s-test-infra
2022/10/04 20:03:09 No matching credentials were found for "us.gcr.io"
Error: GET https://us.gcr.io/v2/token?scope=repository%3Ak8s-test-infra%3Apull&service=us.gcr.io: UNKNOWN: Project 'project:k8s-test-infra' not found or deleted.
```

Reply:
@upodroid this should be `k8s-infra-kubernetes-io` perhaps?

Reply:
No, it needs to be `*.gcr.io/k8s-artifacts-prod/*/foobar`

Reply:
I think these images are not being promoted currently, only pushed to gcr.io/k8s-staging-test-infra

Reply:
I know, there are others as well.

https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io#creating-staging-repos these instructions need to be reconsidered. Everyone is following them even if they don't intend on promoting images from staging.

Reply:
+1 test-infra